cmd/util/cmd/diff-states/cmd.go

```diff
@@ -669,3 +701,29 @@ func (e countDiff) MarshalJSON() ([]byte, error) {
 		State2: e.State2,
 	})
 }
+
+func isRandomBeaconHistory(serviceAccountAddress, address common.Address, domain common.StorageDomain, key any) bool {
+	if serviceAccountAddress.Compare(address) != 0 {
+		return false
+	}
+
+	if domain != randomBeaconHistoryDomain {
+		return false
+	}
+
+	switch key := key.(type) {
+	case interpreter.StringAtreeValue:
+		return interpreter.StringStorageMapKey(key) == randomBeaconHistoryDomainKey
+
+	case interpreter.StringStorageMapKey:
+		return key == randomBeaconHistoryDomainKey
+
+	default:
+		return false
+	}
+}
+
+func serviceAccountAddressForChain(chainID flow.ChainID) flow.Address {
+	sc := systemcontracts.SystemContractsForChain(chainID)
+	return sc.FlowServiceAccount.Address
+}
```

Comment:
Maybe instead of assuming the random beacon history is at the service address, use the address of the random beacon history contract (`SystemContracts.RandomBeaconHistory`), just in case it might change at some point

Reply:
Good idea!  Done in PR #7016 because this PR got auto-merged.